From 2a8726479cea553bf9cca4d1092dc7c1856b2745 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Marek=20Marczykowski-G=C3=B3recki?=
 <marmarek@invisiblethingslab.com>
Date: Tue, 27 Dec 2022 15:49:46 +0100
Subject: [PATCH 5/8] sound: Disable SG-buffer

This appears to be broken under Xen PV (or other Qubes-specific
factor?), so disable it until fixed.
---
 sound/core/Kconfig | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/sound/core/Kconfig b/sound/core/Kconfig
index 48db44fa56fe..789f685d850d 100644
--- a/sound/core/Kconfig
+++ b/sound/core/Kconfig
@@ -259,8 +259,8 @@ config SND_VMASTER
 	bool
 
 config SND_DMA_SGBUF
-	def_bool y
-	depends on X86
+	def_bool n
+#	depends on X86
 
 config SND_CTL_LED
 	tristate
-- 
2.50.1

